.deploy/prod-fss-default.yaml

```diff
@@ -44,6 +44,8 @@ spec:
         kvPath: serviceuser/data/prod/srvfpabakus
       - mountPath: /var/run/secrets/nais.io/appdynamics
         kvPath: /apikey/appdynamics/prod
+      - mountPath: /var/run/secrets/nais.io/azuread
```

Comment:
Dette skal tilgjengeliggjøre  `AZURE_CLIENT_ID` og `AZURE_CLIENT_SECRET` når de starter på nais. Ettersom fp-abakus er lagt inn her; https://github.com/navikt/aad-iac/commit/a3ec21a8c5d77978c5312b023a3135d01dc5339d

Reply:
Husker du hvor man bestiller disse trygdeetaten identer?

Reply:
tror du må sende mail til nav.it.identhandtering@nav.no med din personalleder på kopi og spørre om det som står her;
https://github.com/navikt/aad-iac#owners